apps/charterafrica/src/payload/collections/Tags.js

```diff
@@ -17,7 +17,30 @@ const Tags = {
       localized: true,
       required: true,
     },
-    slug(),
```

Comment:
We messed up here: `slug` should be `tags`. We can't just change it in code because there is already data in production using `tag` slug.

We need to come up with a plan on how to migrate this field. cc @thepsalmist